docs/en/docs/network/modules/egressgateway/create-egpolicy.md

```diff
@@ -73,11 +86,17 @@ Currently, EgressPolicy is divided into two categories: namespace-level policies
 
     In the above creation command:
 
-    * __spec.egressGatewayName__ specifies which EgressGateway group to use.
-    * __spec.appliedTo.podSelector__ specifies which Pods within the cluster this policy will apply to.
+    * `spec.egressGatewayName` specifies which EgressGateway group to use.
+    * `spec.appliedTo.podSelector` specifies which pods within the cluster this policy will apply to.
     * There are two options for the source IP address of egress traffic:
-        * You can use the IP of the gateway node. This is suitable for public cloud and traditional network environments, but the drawback is that the outbound source IP may change with node failures. You can set __spec.egressIP.useNodeIP=true__ to make it effective.
-        * You can use a separate VIP. Since EgressGateway operates on the basis of ARP, it is suitable for traditional networks, but not suitable for public cloud environments. The advantage is that the outbound source IP is always fixed. If no settings are made in EgressPolicy, the default VIP of `egressGatewayName` will be used, or you can manually specify __spec.egressIP.ipv4__, and the IP value must be within the IP pool of the EgressGateway.
+        * You can use the IP of the gateway node. This is suitable for public cloud and traditional
+        network environments, but the drawback is that the outbound source IP may change with node failures.
+        You can set `spec.egressIP.useNodeIP=true` to make it effective.
+        * You can use a separate VIP. Since EgressGateway operates on the basis of ARP,
+        it is suitable for traditional networks, but not suitable for public cloud environments.
+        The advantage is that the outbound source IP is always fixed. If no settings are made in EgressPolicy,
+        the default VIP of `egressGatewayName` will be used, or you can manually specify `spec.egressIP.ipv4`,
+        and the IP value must be within the IP pool of the EgressGateway.
```

Comment:
```suggestion
        * You can use the IP of the gateway node. This is suitable for public cloud and traditional
          network environments, but the drawback is that the outbound source IP may change with node failures.
          You can set `spec.egressIP.useNodeIP=true` to make it effective.
        * You can use a separate VIP. Since EgressGateway operates on the basis of ARP,
          it is suitable for traditional networks, but not suitable for public cloud environments.
          The advantage is that the outbound source IP is always fixed. If no settings are made in EgressPolicy,
          the default VIP of `egressGatewayName` will be used, or you can manually specify `spec.egressIP.ipv4`,
          and the IP value must be within the IP pool of the EgressGateway.
```
折行之后也要对齐